etc/calico/confd/templates/bird6_aggr.cfg.template

```diff
@@ -0,0 +1,33 @@
+# Generated by confd
+# ------------- Static black hole addresses -------------
+{{- $block_key := printf "/calico/ipam/v2/host/%s/ipv6/block" (getenv "NODENAME")}}
+{{if ls $block_key}}
```

Comment:
Just a nit, but it would be nice to add the `-` here to remove the leading spaces and newlines, so the generated template is a bit easier to read.

Reply:
I'd like to do that in a separate PR, to preserve the fact that this PR doesn't change the expected configs for existing tests.

Reply:
Definitely a separate PR if we want to do that. Need to think about it, but it could be nice.